pkg/pod/pods.go

```diff
@@ -711,6 +712,62 @@ func (p *Pod) getModTime(path string) (time.Time, error) {
 	return fi.ModTime(), nil
 }
 
+// getCreationTime returns the time when the pod was created.
+// This happens at prepare time.
+func (p *Pod) getCreationTime() (time.Time, error) {
+	if p.isPrepared || p.isRunning() || p.AfterRun() {
+		return p.getModTime("pod")
+	}
+	return time.Time{}, nil
+}
+
+// getStartTime returns the time when the pod was started.
+func (p *Pod) getStartTime() (time.Time, error) {
+	var (
+		t   time.Time
+		err error
+	)
+	if p.isRunning() || p.AfterRun() {
+		// check pid and ppid files, since stage1 implementations can choose
+		// which one to implement.
+		t, err = p.getModTime("pid")
+		if os.IsNotExist(err) {
+			t, err = p.getModTime("ppid")
+			// if there's an error starting the pod, it can be "exited" without
+			// the "ppid" (or "pid") files being created, return an error only
+			// if it's different than ENOENT.
+			if os.IsNotExist(err) {
+				err = nil
+			}
+		}
+	}
+	return t, err
+}
+
+// GCMarkedTime returns the time when the pod is marked by gc.
+func (p *Pod) GCMarkedTime() (time.Time, error) {
+	if !p.isGarbage && !p.isExitedGarbage {
+		return time.Time{}, nil
+	}
+
+	// At this point, the pod is in either exited-garbage dir, garbage dir or gone already.
+	podPath := p.Path()
+	if podPath == "" {
+		// Pod is gone.
+		return time.Time{}, nil
+	}
+
+	fi, err := os.Lstat(podPath)
+	if err != nil {
+		if err == syscall.ENOENT {
+			// Pod is gone.
+			err = nil
+		}
+		return time.Time{}, err
+	}
+	return fileutil.GetCtime(fi), nil
+}
+
```

Comment:
I think most of this snippet is a leftover from a previous iteration, and you just need the `fileutil.GetCtime` change in here. Can you please double-check?